package.json

```diff
@@ -76,6 +76,7 @@
     "p-map": "^4.0.0",
     "path-exists": "^4.0.0",
     "precinct": "^9.0.1",
+    "pretty-ms": "^7.0.0",
```

Comment:
You can also use v8 now

Reply:
v8 doesn't work with ZISI right now because it's not a module and converting it to one at this time would require a host of lots of other changes.

Reply:
Sorry! Just seen the ESM work, will move to v8, thank you!